Phil Than

pthan1 opened the pull request.

francepack reviewed (commented):
@pthan1 Overall, nice work here. Please see the note made below about the calendar exercise, and add those weeks of your calendar as soon as you can. 

On the technical side of things, your work is looking good- nice attention to detail throughout.

Please look over this feedback, and you will be good to go once you have added in the requested weeks of your calendar. Message me here when those are in.

pthan1 commented:
Hi Mason,

Thanks for the feedback.  I will take these and incorporate them into my
work.

I have added Week 1-3 Calendar screenshots and updated my letter to my
mentor.

Thanks,
Phil

On Wed, Jun 16, 2021 at 11:08 AM Mason France ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
>
> @pthan1 <https://github.com/pthan1> Overall, nice work here. Please see
> the note made below about the calendar exercise, and add those weeks of
> your calendar as soon as you can.
>
> On the technical side of things, your work is looking good- nice attention
> to detail throughout.
>
> Please look over this feedback, and you will be good to go once you have
> added in the requested weeks of your calendar. Message me here when those
> are in.
> ------------------------------
>
> In finalPrep/README.md
> <https://github.com/turingschool/frontend-mod-1-prework/pull/115#discussion_r652830784>
> :
>
> > @@ -31,24 +32,34 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
>
>  When you are finished, add screenshots of your calendar so we can provide feedback if needed!
>
> -- `Add Week 1 Screenshot Here`
> -- `Add Week 2 Screenshot Here`
> -- `Add Week 3 Screenshot Here`
> +- `Add Week 1 Screenshot Here`- * Turing Mod 1 / Week 1 Calendar for May already passed and overlapped with previous Calendar items.  Similar to Week 3.*
> +- `Add Week 2 Screenshot Here` - * Turing Mod 1 / Week 1 Calendar for May already passed and overlapped with previous Calendar items. Similar to Week 3.*
> +- `Add Week 3 Screenshot Here`<img width="1242" alt="Screen Shot 2021-05-29 at 9 09 46 PM" src="https://user-images.githubusercontent.com/82873669/120091197-18f99700-c0c6-11eb-9f35-4dda63e92fac.png">
>
> We were looking for the first 3 weeks of your Mod1 here. So, looking ahead
> to when you are enrolled at Turing. It looks like you have a good idea of
> some events that are recurring, and have connected to the Mod1 calendar
> already, so you already have good pieces in place to get your calendar
> together for those weeks. As you look at your calendar for those weeks, I
> recommend adding a block on Fridays to review your past week, and update
> your upcoming week's calendar. Also, try to anticipate what kinds of work
> time you'll need on the weekends, as well as making sure you have personal
> time blocked off.
> ------------------------------
>
> In finalPrep/annotations.js
> <https://github.com/turingschool/frontend-mod-1-prework/pull/115#discussion_r652851106>
> :
>
> >    var builtBear = {
>      basicInfo: demographics,
>      clothes: clothes,
>      exterior: fur,
>      cost: 49.99,
>      sayings: [greeting, powerSaying, "Goodnight my friend!"],
> -    isCuddly: true,
> +    isCuddly: true, //Assign a variable named builtBear to an object with parameters: basicInfo (string), clothes (string), exterior (string), cost (float), sayings (array), isCuddly (boolean).
>
> Nice work overall with your annotations. I think you could try to be more
> specific with where variables come from. For example, is it a parameter, or
> was it declared above. basicInfo has a property of demographics, which is
> not a string- an example of why it can be important to keep track of where
> variables come from.
> ------------------------------
>
> In finalPrep/modZeroHero.js
> <https://github.com/turingschool/frontend-mod-1-prework/pull/115#discussion_r652854261>
> :
>
> >  // - Create 2 instances of your SuperHero class
> +var batman = new SuperHero('Batman', 'fear', 35)
> +console.log(batman);
> +batman.sayName();
> +batman.maximizeEnergy();
> +batman.gainPower(3);
> +
> +console.log(batman);
> +var robin = new SuperHero('Robin', 'one-man-army', 29)
> +console.log(robin);
>
> Nice job with all of this!
> ------------------------------
>
> In section4/exercises/classMethods.js
> <https://github.com/turingschool/frontend-mod-1-prework/pull/115#discussion_r652868218>
> :
>
> >    }
>
> -  // ADD CODE
> -};
> +  changeProtein(protein2) {
> +    this.protein = protein2;
> +  }
> +
> +  addTopping(newTopping) {
> +    this.toppings.push(newTopping);
> +  }
>
> Nice work with these dynamic functions
> ------------------------------
>
> In finalPrep/README.md
> <https://github.com/turingschool/frontend-mod-1-prework/pull/115#discussion_r652878451>
> :
>
> >
>  ### Mentorship Prep
>  Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
>  - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
> -  - [ ] Add link to your gist here:
> +<<<<<<< HEAD
> +  - [ ] Add link to your gist here:
> +[Mentorship Prep Gist](https://gist.github.com/pthan1/dbdf2151e0262f7da6c90c3f8575bc7f)
> +=======
>
> Good work here. I encourage you to add some detail in your letter about
> why you are excited about becoming a developer, perhaps some goals you have
> for Mod1, and how you hope to work together.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/turingschool/frontend-mod-1-prework/pull/115#pullrequestreview-685365968>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATYI2RKUPPTICTI4QNOJS2TTTDK77ANCNFSM45Y6MWEA>
> .
>


-- 
Philong Than
